### ISSUE TITLE
remove_outliers: return_mask defaults to False, but is documented as defaulting to True

### ISSUE THREAD

**Issue by ceb8** [KeplerGO/lightkurve#529]
#### Problem description
In the docstring of `LightCurve.remove_outliers`, the argument `return_mask` is claimed to default to `True`, however is that argument is not explicitly set, the mask array is not returned. 

#### Example
```python
import lightkurve

lc_list = lightkurve.search_lightcurvefile(9592705, quarter=9, cadence='short').download_all()

lc_fle = lc_list[0]
sapflux = lc_fle.SAP_FLUX
    
clipped_flux = sapflux.remove_outliers(sigma=3.5)
```

#### Expected behavior
```python
import lightkurve

lc_list = lightkurve.search_lightcurvefile(9592705, quarter=9, cadence='short').download_all()

lc_fle = lc_list[0]
sapflux = lc_fle.SAP_FLUX
    
clipped_flux, mask = sapflux.remove_outliers(sigma=3.5)
```

#### Environment:
- platform (e.g. Linux, OSX, Windows): OSX
- lightkurve version (e.g. 1.0b6): 1.0.0
- installation method (e.g. pip, conda, source): conda

**Comment by barentsen** [comment]
Thank you for reporting this issue @ceb8!  This is a bug!

The default value of `return_mask` is intended to be `False` (because `LightCurve` methods all tend to return a new `LightCurve` by default and nothing else).  The docstring incorrectly claims the default for `return_mask` to be `True`.  I intend to open a PR shortly to correct the docstring!

Thanks++.

**Comment by barentsen** [comment]
The docstring has been fixed as part of #530.  Thanks again for reporting the issue @ceb8!!
